heareval/predictions/task_predictions.py

```diff
@@ -35,28 +49,76 @@ def __init__(self, nfeatures: int, nlabels: int, prediction_type: str):
         torch.nn.init.normal_(self.projection.weight)
         if prediction_type == "multilabel":
             self.activation: torch.nn.Module = torch.nn.Sigmoid()
+            self.logitloss = torch.nn.BCEWithLogitsLoss()
         elif prediction_type == "multiclass":
             self.activation = torch.nn.Softmax()
+            self.logit_loss = OneHotToCrossEntropyLoss()
```

Comment:
Do we need `torch.nn.Softmax()` as activation? We already are using `torch.nn.CrossEntropyLoss()` in the `OneHotToCrossEntropyLoss()`, which does softmax before applying the cross entropy?